common/app/assets/assets.scala

```diff
@@ -1,168 +1,135 @@
-      }
+  private val memoizedCss: ConcurrentMap[String, Try[String]] = TrieMap()
 
-      memoizedSvg.getOrElseUpdate(path, loadFromDisk)
-    }
-  }
+  def head(projectOverride: Option[String]) = inline(cssHead(projectOverride.getOrElse(Configuration.environment.projectName)))
+  def inlineStoryPackage = inline("story-package")
 
-  object css {
+  def projectCss(projectOverride: Option[String]) = project(projectOverride.getOrElse(Configuration.environment.projectName))
+  def headOldIE(projectOverride: Option[String]) = cssOldIE(projectOverride.getOrElse(Configuration.environment.projectName))
+  def headIE9(projectOverride: Option[String]) = cssIE9(projectOverride.getOrElse(Configuration.environment.projectName))
 
-    private val memoizedCss: ConcurrentMap[java.net.URL, String] = TrieMap()
-
-    def projectCss(projectOverride: Option[String]) = project(projectOverride.getOrElse(Configuration.environment.projectName))
-    def head(projectOverride: Option[String]) = cssHead(projectOverride.getOrElse(Configuration.environment.projectName))
-    def headOldIE(projectOverride: Option[String]) = cssOldIE(projectOverride.getOrElse(Configuration.environment.projectName))
-    def headIE9(projectOverride: Option[String]) = cssIE9(projectOverride.getOrElse(Configuration.environment.projectName))
+  private def inline(module: String): String = {
+    val resourceName = s"assets/inline-stylesheets/$module.css"
+    Get(if (Play.current.mode == Mode.Dev) {
+      LoadFromClasspath(resourceName)
+    } else {
+      memoizedCss.getOrElseUpdate(resourceName, LoadFromClasspath(resourceName))
+    })
+  }
 
-    def inline(module: String): Option[String] = {
-       val knownInlines : PartialFunction[String,String] =
-       {
-         case "story-package" => "story-package.css"
-       }
-       knownInlines.lift(module).map { cssModule => loadCssResource(s"assets/inline-stylesheets/$cssModule") }
+  private def project(project: String): String = {
+    project match {
+      case "facia" => "stylesheets/facia.css"
+      case _ => "stylesheets/content.css"
     }
+  }
 
-    private def cssHead(project: String): String = {
-
-      val suffix = project match {
-        case "footballSnaps" => "footballSnaps.css"
-        case "facia" => "facia.css"
-        case "identity" => "identity.css"
-        case "football" => "football.css"
-        case "index" => "index.css"
-        case "rich-links" => "rich-links.css"
-        case "email" => "email.css"
-        case _ => "content.css"
-      }
-
-      loadCssResource(s"assets/inline-stylesheets/head.$suffix")
+  private def cssHead(project: String): String =
+    project match {
+      case "footballSnaps" => "head.footballSnaps"
+      case "facia" => "head.facia"
+      case "identity" => "head.identity"
+      case "football" => "head.football"
+      case "index" => "head.index"
+      case "rich-links" => "head.rich-links"
+      case "email" => "head.email"
+      case _ => "head.content"
```

Comment:
nitpick: to avoid repetition why not:
`
val cssHeads = List("footballSnaps", "facia", ..., email)
val headSuffix = if (cssHeads.contains(project)) project else "content"
s"head.${headSuffix}"
`